OpenRA.Game/World.cs

```diff
@@ -68,15 +68,19 @@ public void SetPlayers(IEnumerable<Player> players, Player localPlayer)
 		public Player LocalPlayer { get; private set; }
 
 		public event Action GameOver = () => { };
+
+		/// <Remarks> Should only be set in <see cref="EndGame"/></Remarks>
 		public bool IsGameOver { get; private set; }
 		public void EndGame()
 		{
 			if (!IsGameOver)
 			{
+				SetPauseState(true);
```

Comment:
Is there a reason you put this at the start of the method instead of at the end (which matches the previous behaviour)?

Reply:
Yes, because I made it so you can't change PausedState when IsGameOver is true

Reply:
Ah, makes sense.